docs/docs/tutorials/getting-started-guide.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: Getting Started Guide
+---
+
+# Getting Started Guide
+
+This guide will walk you through creating your first project in LiveCodes.
+
+## Prerequisites
+
+- A web browser (Chrome, Firefox, Safari, or Edge)
+- Basic knowledge of HTML, CSS, and JavaScript
+
+## Step 1: Open LiveCodes
+
+1. Navigate to [livecodes.io](https://livecodes.io)
+2. You'll see the editor interface with three panels:
+   - **HTML** (left)
+   - **CSS** (middle)
+   - **JavaScript** (right)
+   - **Result** (bottom)
+
+## Step 2: Create a Simple Page
+
+Let's create a simple interactive greeting card.
+
+### HTML Panel
+```html
+<div class="container">
+  <h1 id="greeting">Hello, World!</h1>
+  <input type="text" id="nameInput" placeholder="Enter your name">
+  <button id="greetBtn">Greet Me!</button>
+</div>
+```
+
+### CSS Panel
+```css
+.container {
+  max-width: 600px;
+  margin: 50px auto;
+  text-align: center;
+  font-family: Arial, sans-serif;
+}
+
+h1 {
+  color: #007bff;
+  font-size: 2.5rem;
+}
+
+input {
+  padding: 10px;
+  font-size: 1rem;
+  margin: 10px;
+  border: 2px solid #007bff;
+  border-radius: 5px;
+}
+
+button {
+  padding: 10px 20px;
+  font-size: 1rem;
+  background-color: #007bff;
+  color: white;
+  border: none;
+  border-radius: 5px;
+  cursor: pointer;
+}
+
+button:hover {
+  background-color: #0056b3;
+}
+```
+
+### JavaScript Panel
+```javascript
+const greeting = document.getElementById('greeting');
+const nameInput = document.getElementById('nameInput');
+const greetBtn = document.getElementById('greetBtn');
+
+greetBtn.addEventListener('click', () => {
+  const name = nameInput.value.trim();
+  if (name) {
+    greeting.textContent = `Hello, ${name}! `;
+  } else {
+    greeting.textContent = 'Hello, World!';
+  }
+});
+```
+
+## Step 3: See Your Results
+
+The result panel automatically updates as you type. Try:
+- Entering your name in the input field
+- Clicking the "Greet Me!" button
+- Modifying the colors in the CSS
+
+## Step 4: Save Your Project
+
+1. Click the "Share" button in the toolbar.
+2. Choose **"Save"** to create a permanent URL
+3. Bookmark or share your project URL
+
```

Comment:
It would be nice to add here a link for the completed project (e.g. https://livecodes.io/?x=id/q6k8itvp2dv)